src/main/update/index.ts

```diff
@@ -79,13 +80,7 @@ async function getUpdater(): Promise<AppUpdater | undefined> {
   let updater: AppUpdater;
 
   try {
-    let appUpdateConfigPath: string;
-
-    if (Electron.app.isPackaged) {
-      appUpdateConfigPath = path.join(process.resourcesPath, 'app-update.yml');
-    } else {
-      appUpdateConfigPath = path.join(Electron.app.getAppPath(), 'dev-app-update.yml');
-    }
+    const appUpdateConfigPath = resources.get('app-update.yml');
```

Comment:
This is trying to duplicate the code that [electron-updater](https://www.electron.build/auto-update#debugging) actually does; wouldn't just changing it mean we'll diverge (and therefore break things)?

Reply:
Oh I wasn't aware it was trying to do that. Either way, I think it should be fine as long as we are able to load the info in app-update.yml alright and pass it as options to whichever implementation of `AppUpdater` we're using. If you look at the `CustomPublishOptions` interface you can see that there are not any real constraints on the shape of this options object.